src/externalized/strings/ItemStrings.cc

```diff
@@ -1,58 +1,72 @@
-#include "ItemStrings.h"
-#include "EncryptedString.h"
 #include "Directories.h"
+#include "EncryptedString.h"
+#include "GameRes.h"
+#include "ItemStrings.h"
+#include "Tactical_Save.h"
 
 #define VANILLA_MAX_ITEMS    351
 #define SIZE_ITEM_NAME        80
 #define SIZE_SHORT_ITEM_NAME  80
 #define SIZE_ITEM_INFO       240
 
-VanillaItemStrings::VanillaItemStrings() {}
+BinaryData::BinaryData() {}
 
 // Needed to be able to return a reference to an empty string which is not local
 const ST::string EMPTY_STRING = "";
+// This is to avoid failures due to prof.dat unavailability during unit-testing
+static const MERCPROFILESTRUCT* DEFAULT_PROFILE = new MERCPROFILESTRUCT();
```

Comment:
There's no need to heap allocate this and I'd prefer if you defined this in getProfile, the only place where it is used. That would also explain the the mProfiles.size() check in that function which ATM is a bit obscure.